js/fieldmanager.js

```diff
@@ -267,4 +267,65 @@ $( document ).ready( function () {
 	$( document ).on( 'fm_activate_tab', init_sortable );
 } );
 
+/**
+ * Add character counter to input text / textarea elements
+ *
+ * @param object
+ *  - selector string jQuery selector
+ *  - type string It ca be 'lenght' (displays typed chars) or 'maxlength' (displays remaining chars)
+ */
+fm_add_character_counter = function() {
+	var args = jQuery.extend( {
+		debug: 0,
+		selector: null,
+		type: 'maxlength',
+		maxlength: 100,
+		force_container_block: false
+	}, arguments[0] );
+
+	jQuery( args.selector ).each( function(){
+
+		var el = jQuery( this );
+
+		// Check for fm
+		if ( el.closest( '.fm-item' ).hasClass( 'fmjs-proto' ) ) {
+			return;
+		}
+
+		// Add padding
+		el.addClass( 'fm-character-counter-padding' );
+
+		// Check type of counter
+		var type = typeof( el.attr( 'maxlength' ) ) === 'undefined' ? 'length' : args.type;
+
+		var div = jQuery( '<div />' );
+		var span = jQuery( '<span />' );
+
+		// Fix for textarea
+		if ( /^textarea$/i.test( el.prop( 'tagName' ) ) ) {
+			span.css( 'top', 'inherit' );
```

Comment:
Yes, of course.
The default CSS values were for demonstration purposes and can be improved/modified as the inline JS fixes. I just committed a fix to move them to the CSS file.
FYI the `display: block;` is for WP textarea fields.